torchreid/engine/image/multilabel.py

```diff
@@ -44,7 +45,13 @@ def __init__(self, datamanager, models, optimizers, schedulers, use_gpu, save_al
         if not isinstance(num_classes, (list, tuple)):
             num_classes = [num_classes]
         self.num_classes = num_classes
-        self.scale = s
+        self.scales = dict()
```

Comment:
May be this code should be extracted to the Engine class, since it's used by more than one subclass (something like collect_scales() method)

Reply:
agree